### ISSUE TITLE
Improve default line spacing and UX of line spacing preferences

### ISSUE THREAD

**Issue by sloria** [glushchenko/fsnotes#887]
**Describe your feature request**

In priority order:

1. The default line spacing (0) is too cramped to be very readable.
2. A slider (screenshot below) isn't an intuitive control for line spacing. A menu with "1", "1.5", "2" etc. would be easier to use and would allow me to more easily match my configurations across machines.
3. I expected to be able to configure the line spacing in the "Layout / Font" section of the Preferences. It wasn't until I found https://github.com/glushchenko/fsnotes/issues/262 that I found that the settings are in "Editor / Line Spacing".

![Image 2020-05-11 at 10 39 56 AM](https://user-images.githubusercontent.com/2379650/81574424-c858b400-9373-11ea-98b8-b5fec1a0f00a.png)

**Additional context**
macOS

**Comment by sloria** [comment]
Re: 2, Bear also uses a slider for line spacing:

<img width="482" alt="Image 2020-05-11 at 10 47 13 PM" src="https://user-images.githubusercontent.com/2379650/81632967-8eba9400-93d9-11ea-9d8c-ab48c724b8e3.png">

This still seems like an odd choice, but maybe it's fine to follow Bear's lead for now.

Re: 3, moving the "Note Font" (and possibly rename to just "Font") setting to the "Editor" tab would address the issue. Font, line spacing, and line width settings should be colocated.

**Comment by sloria** [comment]
It would also be good to have the "Reset to defaults" button, like in the Bear screenshot above. As a user, I expect that option when the controls are sliders.

**Comment by brianwood2783** [comment]
I agree. I think the slider can be maintained, but it could have "notches" like regular menus on macOS, for example when setting the trackpad sensibility:

![image](https://user-images.githubusercontent.com/65142587/82156395-67951400-986a-11ea-9c51-182643f7d31f.png)

I guess 7 or 8 notches would be sufficient (as in the scrolling speed). Another related issue is that as of 4.3.3, the note titles aren't perfectly centralized height-wise... could be because of the slider.

**Comment by glushchenko** [comment]
https://github.com/glushchenko/fsnotes/releases/tag/7.0.0b1
